app/javascript/chat/chat.jsx

```diff
@@ -1409,7 +1410,35 @@ export default class Chat extends Component {
       .getElementById('jumpback_button')
       .classList.remove('chatchanneljumpback__hide');
   };
-
+  handleImageDrop = (e) => {
+    document.getElementById('messagelist').style.opacity = 1;
+    dragDrop(e, this.handleImageSuccess, this.handleImageFailure);
+  };
+  handleImageSuccess = (res) => {
+    const mLink = `![alt text](${res.links[0]})`;
+    const el = document.getElementById('messageform');
+    const start = el.selectionStart;
+    const end = el.selectionEnd;
+    const text = el.value;
+    let before = text.substring(0, start);
+    before = text.substring(0, before.lastIndexOf('@') + 1);
+    const after = text.substring(end, text.length);
+    el.value = `${before + mLink} ${after}`;
+    el.selectionStart = start + mLink.length + 1;
+    el.selectionEnd = start + mLink.length + 1;
+    el.focus();
+  };
+  handleImageFailure = (e) => {
+    addSnackbarItem({ message: e.message, addCloseButton: true });
```

Comment:
Snackbar! 😎

Reply:
?

Reply:
I just meant that you were using the `<Snackbar />` component. 😄